CONFIGURE.md

```diff
@@ -287,6 +341,62 @@ Another option to define a state storage is to use the following properties in t
     nifi.state.storage.local.auto.persistence.interval=0 sec
 
 
+### Configuring Repositories
+
+Apache MiNiFi C++ uses three repositories similarily to Apache NiFi:
+- The Flow File Repository holds the metadata (like flow file attributes) of the flow Files.
+- The Content Repository holds the content of the flow files.
+- The Provenance Repository holds the history of the flow files.
+
+The underlying implementation to use for these repositories can be configured in the minifi.properties file.
+
+The Flow File Repository can be configured with the `nifi.flowfile.repository.class.name` property. If not specified, it uses the `FlowFileRepository` class by default, which stores the flow file metadata in a RocksDB database. Alternatively it can be configured to use a `VolatileFlowFileRepository` that keeps the state in memory and flushes upon restart, or the `NoOpRepository` for not keeping any state.
```

Comment:
I think "so the state gets lost" would be clearer than "and flushes"  ("flushes" suggests writing it out somewhere)